xml/rmt_mirroring.xml

```diff
@@ -4,46 +4,43 @@
   <!ENTITY % entities SYSTEM "entity-decl.ent">
     %entities;
 ]>
-
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-rmt-mirroring">
 <!-- ==================================================================== -->
  <title>Mirroring repositories on the &rmt; server</title>
  <info/>
  <para>
-  You can mirror the installation and update repositories on the &rmt;
-  server. This way, you do not need to download updates on each
-  machine, which saves time and bandwidth.
+  You can mirror the installation and update repositories on the &rmt; server.
+  This way, you do not need to download updates on each machine, which saves
+  time and bandwidth.
  </para>
  <para>
-  In its default configuration, &rmt; mirrors enabled product
-  repositories automatically once every night.
+  In its default configuration, &rmt; mirrors enabled product repositories
+  automatically once every night.
  </para>
  <para>
-   By default, the mirrored repositories are stored in
-   <filename>/var/lib/rmt/public/repo</filename>.
+  By default, the mirrored repositories are stored in
+  <filename>/var/lib/rmt/public/repo</filename>.
  </para>
- 
  <note>
-   <title>Change default location of the mirrored repositories</title>
-   <para>
-     To change the default location of the mirrored repositories,
-     point the <filename>/usr/share/rmt/public/repo</filename>
-     symbolic link to the desired directory. This can be done using
-     the <command>ln -sfn <replaceable>TARGET</replaceable> /usr/share/rmt/public/repo
-     </command> command (replace
-     <literal>TARGET</literal> with the desired destination). Make
-     sure that the target must have read and write permissions for the
-     <literal>rmt</literal> user and <literal>nginx</literal> group.
-     If you prefer to store mirrored repositories in a different
-     directory, or on a different storage device,
-   </para>
-   </note>
+  <title>Change default location of the mirrored repositories</title>
+  <para>
+   To change the default location of the mirrored repositories, point the
+   <filename>/usr/share/rmt/public/repo</filename> symbolic link to the desired
+   directory. This can be done using the <command>ln -sfn
+   <replaceable>TARGET</replaceable> /usr/share/rmt/public/repo </command>
+   command (replace <literal>TARGET</literal> with the desired destination).
+   Make sure that the target must have read and write permissions for the
+   <literal>rmt</literal> user and <literal>nginx</literal> group. If you
+   prefer to store mirrored repositories in a different directory, or on a
+   different storage device,
```

Comment:
Comma at the end, the sentence is unfinished ... I guess you may want to fix that here too..?
Last changes:

```
e76e475b4b xml/rmt_mirroring.xml (Dmitri Popov       2020-05-20 13:07:27
```

Reply:
I have looked at the 15.2 docs. It is the same there. The paragraph is not in the 15.1 docs. I do not know how to follow your reference. :-(

Reply:
Oh, my wording was a little bad. It was an extract from a `git blame` run  on the file.

Essentially, @dmpop was the person who added the unfinished sentence in May 2020 in commit e76e475.

Reply:
This looks like a rogue sentence. Just remove _If you prefer to store mirrored repositories in a different
     directory, or on a different storage device,_ Thank you and sorry!

Reply:
Thanks! Fixed. I also put the same command on its own line and split off the run-on sentence.